popup.html

```diff
@@ -13,6 +13,8 @@
       integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
       crossorigin="anonymous"
     />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.13.0/css/all.css">
```

Comment:
By custom icons i mean that they should be in the assets folder so that there is no need to load them from font-awesome

Reply:
okay :)